gulpfile.js

```diff
@@ -67,12 +64,11 @@ async function html() {
 const rollupCaches = new Map();
 
 async function js(entry, outputPath) {
-  const parsedPath = path.parse(entry);
-  const name = /[^/]+$/.exec(parsedPath.dir)[0];
-  const changelog = await readJSON(`${__dirname}/src/changelog.json`);
+  const name = path.basename(path.dirname(entry));
+  const changelog = await readJSON(path.join(__dirname, 'src/changelog.json'));
   const bundle = await rollup.rollup({
     cache: rollupCaches.get(entry),
-    input: `src/${entry}`,
+    input: path.normalize(`src/${entry}`),
```

Comment:
Doesn't rollup handle this already?

Reply:
Probably, yeah. I can revert this and the `file` change.